src/arrayMethodIncludes.js

```diff
@@ -4,8 +4,23 @@
  * Implement method includes
  */
 function applyCustomIncludes() {
-  [].__proto__.includes2 = function(valueToFind, fromIndex) {
-    // write code here
+  [].__proto__.includes2 = function(valueToFind, fromIndex = 0) {
+    const arrayLength = this.length;
```

Comment:
use it everywhere where need this.length or just remove

Reply:
done( completely forgot about this